de.peeeq.wurstscript/src/main/java/de/peeeq/wurstscript/validation/TRVEHelper.java

```diff
@@ -0,0 +1,7 @@
+package de.peeeq.wurstscript.validation;
+
+import java.util.HashSet;
+
+public class TRVEHelper {
+    public static final HashSet<String> protectedVariables = new HashSet<>();
```

Comment:
Please don't use static variables with mutable state.

Reply:
You didn't answer last time. I can't add it as variable to the ImTranslator because I need it in the validator. You want it in the outer scope of that?